### PR TITLE
{tools}[GCC/11.2.0] hector v2.5.0

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

Requires:
- [x] PR #15384

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/h/hector/hector-2.5.0-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    (['src/libhector.a'], 'lib'),
+    'R', 'man', 'LICENSE.md', 'README.md', 'DESCRIPTION', 'scripts', 'data-raw', 'inst',
+]
+
+sanity_check_paths = {
+    'files': ['bin/hector'],
+    'dirs': [],
+}
+
+moduleclass = 'tools'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/h/hector/hector-2.5.0-GCC-11.2.0.eb:40]:
```suggestion
moduleclass = 'geo'
```
?

Comment (sassy-crick) [reply easybuild/easyconfigs/h/hector/hector-2.5.0-GCC-11.2.0.eb:40]: Looks better to me. I was not sure what to choose so I went for `tools`. I have updated that to `geo`. Thanks!

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**